src/app/npg-lite/page.tsx

```
@@ -935,47 +959,50 @@ const NPG_Ble = () => {
+        // Process any remaining data in the buffer
+        if (fillingindex.current > 0) {
+            // Create a copy of the current buffer data
+            const remainingData = recordingBuffers[activeBufferIndex].slice(0, fillingindex.current);
+            recordingBuffers[activeBufferIndex] = remainingData;
+
+            // Process the remaining buffer
+            await processBuffer(activeBufferIndex, canvasElementCountRef.current, selectedChannels);
+        }
+
+        // Clear buffers after processing
+        recordingBuffers.forEach(buffer => buffer.length = 0);
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)